retis-events/src/ovs.rs

```diff
@@ -367,6 +369,8 @@ impl EventFmt for ActionEvent {
             None => write!(f, " unspec")?,
         }
 
+        write!(f, " nla {:x}", self.action_address)?;
```

Comment:
nit: `:#x`

A suggestion: precision like the above are sometimes put between parenthesis in our events, eg. `(nla {})`.

Reply:
if we want to stay consistent (with e.g. skb) parentheses are wanted, but the prefixed hex is not what we want.